web/client-api/src/test/java/io/deephaven/web/client/api/TotalsTableTestGwt.java

```diff
@@ -371,6 +374,33 @@ public void testGroupedTotals() {
                 .then(this::finish).catch_(this::report);
     }
 
+    public void testCreateTotalsTableAggTypes() {
+        connect(tables)
+                .then(table("hasTotals"))
+                .then(table -> {
+                    List<Supplier<Promise<JsTotalsTable>>> tests = new ArrayList<>();
+                    List<String> aggs = JsTotalsTableConfig.knownAggTypes;
+                    for (int i = 0; i < aggs.size(); i++) {
+                        String operation = aggs.get(i);
+                        JsTotalsTableConfig config = new JsTotalsTableConfig();
+                        config.groupBy.push("K");
+                        config.operationMap.set("J",
+                                Js.uncheckedCast(new JsString[] {toJsString(operation)}));
+                        tests.add(() -> table.getTotalsTable(config).then(totals -> {
+                            totals.setViewport(0, 100, null, null, null);
+
+                            return waitForEvent(totals, JsTable.EVENT_UPDATED, update -> {
+                                ViewportData viewportData = (ViewportData) update.getDetail();
+                                assertEquals(2, viewportData.getRows().length);
```

Comment:
Should probably include checks for the content of the cells in these rows, similar to the above test cases.

Reply:
Colin asked for a test similar to `HierarchicalTableTestGwt.testCreateRollupAggTypes`, which looks like it iterates over all the agg types and checks that they work and send an update. Checking the data might be complicated because we are doing every agg, so the data will be different per agg.

Reply:
Added some value checking.

Reply:
Data checking in the rollup test doesnt exist either - the intention was just to make sure we could at least read every type, and that no supported agg causes a failure when passing it to the API.